wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+De manière simplifiée, proche dans le temps = proche dans l'espace et loin dans le temps = loin dans l'espace.
+
+Votre espacement des pulsations joue un rôle crucial lorsque le joueur joue à votre beatmap !
+Un espacement constant des pulsations est une condition de base pour ranker votre beatmap, car il procure au joueur une expérience plus intuitive qui lui permet d'apprendre comment bouger son curseur sur l'écran à vitesse constante.
+
+Placer un objet plus proche que l'espacement de pulsation peut tromper le joueur et le faire cliquer sur ces éléments trop tôt.
+D'autre part, le placer trop loin peut forcer le joueur à bouger son curseur à travers l'écran pour cliquer dessus dans les temps.
+Une exception commune pour l'espacement des pulsations sont les cercles de frappe superposés.
+
+Pour assurer un espacement constant, activez le bouton `Distance Snap` sur le côté droit de l'éditeur.
+Vous pouvez choisir une valeur d'espacement en maintenant `ALT` et en slidant le sélecteur situé en haut à droite jusqu'a ce que vous atteignez la valeur désirée.
+Notez qu'une valeur plus élevée donnerait lieu à une beatmap plus difficile.
+
+### Beatmap
+
+Une beatmap (aussi connue comme étant un niveau (level) ou encore une map) est là ou repose la majeure partie d'osu! .
+C'est comme les niveaux dans le jeu DS.
+Dans leurs formes les plus basiques, les beatmaps ont une chanson, des cercles de frappe, des sliders et / ou des spinners plaçés en fonction du rythme de la musique.
+De nos jours, les beatmaps peuvent aussi contenir des images ou vidéos d'arrière-plan, ou encore un storyboard.
+
+Pour plus de détails à propos de l'upload des maps, consultez BSS.
+
+### Beatmappeur
+
+Un beatmappeur (ou mappeur) est un utilisateur qui se sert de l'éditeur pour créer des beatmaps.
+
+### Beatmapset
+
+Un beatmapset (ou mapset) représente l'ensemble des beatmaps qui sont contenues dans un fichier `OSZ` crées par le même beatmappeur.
+Les beatmapsets auront toujours la même musique/son, vidéo, si présente.
+Cependant, merci de noter que certaines beatmaps d'un beatmapset peuvent contenir une image d'arrière-plan et/ou un storyboard spécifique différent des autres.
+
+### BG
+
+BG est l'acronyme pour background.
+BG représente l'image (ou la vidéo) d'arrière-plan d'une beatmap.
+A l'intérieur du dossier d'une beatmap, "BG" est le nom courant pour l'image de fond.
+
+## BN
+
+BN est l'acronyme pour Beatmap Nominator qui est un groupe d'utilisateurs chargés de nominer les beatmaps dites "en attente" (pending beatmaps).
+Ce groupe était précédemment connu sous l'acronyme de BAT (Beatmap Appreciation Team).
+
+Pour plus de détails à propos des Beatmap Nominators, consultez cet article : [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group/).
+
+### BPM
+
+BPM est l'acronyme de Beats Per Minute. Il représente le nombre de pulsations entières en une minute.
+
+### Pause (terme original : Break)
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Pause

_Terme original : Break_
```